cli/cmd/generate.go

```diff
@@ -263,6 +264,11 @@ func injectInitializer(paths []string, imageReplacementsFile string, logger *slo
 		if err != nil {
 			return fmt.Errorf("could not parse image definition file %s: %w", imageReplacementsFile, err)
 		}
+	} else {
+		replacements, err = kuberesource.ImageReplacementsFromFile(bytes.NewReader(releaseImageReplacements))
```

Comment:
This is now always going to fail for dev builds (because of the embedded bogus file). Is that intentional? If not, you could prefix the placeholder text with `#` to just yield no replacements.